index.md

```diff
@@ -0,0 +1,364 @@
+
+## Labels
+
+Labels are used to quickly find tickets, create queries, boards, and dashboards.
+The labels here should all be lower case and words separated by a underscore (`_`).
+Please, avoid creating labels that can be de-coupled into two other labels.
+For example: instead of `lsstcam_science`, use `lsstcam` and `science`.
+Or, even better, just use `science`, since `lsstcam` is the default camera.
+
+| **Label**           | **Description**                                                                 |
+|---------------------|---------------------------------------------------------------------------------|
+| `create`            | Used for the creation of a test case or JSON file.                              |
+| `update`            | Used for updating a test case or JSON file.                                     |
+| `json_block`        | Used when a JSON file is required.                                              |
+| `on_sky/not_on_sky` | Used for on-sky or not-on-sky tests.                                            |
+| `daytime_test`      | Used for tests that could be performed during the daytime.                      |
+| `parallelize`       | Used for tests that have potential for being parallelized.                      |
+|---------------------|---------------------------------------------------------------------------------|
+| `aos_commissioning` | Used for tests associated with the Active Optics System                         |
+| `LSSTCam`           | Used for tests aiming to check functionality of requirements associated with LSSTCam |
+| `calibration`       | Used for tests related to the calibration system                                |
+| `image_quality`     | Used for tests associated with image quality.                                   |
+| `science`           | Used for Science Observations programs                                          |
+| `stray_light`       | Used for tests associated with stray light investigations.                      |
+| `standard_procedures` | Used for procedures that are represented as test cases.                       |
+|-------------------------------------------------------------------------------------------------------|
+
+Each row in the second part of the table above corresponds to a Quick Filter in the [Blocks Creation Status](https://rubinobs.atlassian.net/jira/software/c/projects/BLOCK/boards/259) board.
+
+## Priorities
+
+Each of these steps has a definition of priority.
+We should have an agreement on each of them. Here is the first proposal.
+When it comes to priorities, we must agree on the following points:
+
+We must have 3 to 5 priorities at most. Proposed: **critical** and/or **blocker**, **high**, **standard**, **low**.
+If everything is a top priority, we do not have priorities.
+For each priority, we want to have:
+
+* A clear criteria to help people use the correct priority.
+* Expected estimated time for completion
+
+# Test Planning
```

Comment:
I think you'll find some useful source material in the ConOps draft, which has borrowed a bit from these same confluence pages and workflows we've described. 

See -> https://docs.google.com/document/d/1dILbW_tsbCOueqF73gboVSzuhTlrrnYmXv7NrewqZmc/edit?tab=t.0#heading=h.29vgv89d3ln3